tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime.md

```diff
@@ -72,6 +72,9 @@ This command ensures the runtime compiles without errors, validates the pallet c
 
 Launch your parachain locally and start producing blocks:
 
+!!!info 
+    The chain specification generated above includes "Alice" and "Bob" as pre-funded accounts. Take a look at the [Polkadot.js Accounts section](https://polkadot.js.org/apps/#/accounts){target=\_blank}. These accounts can be used to sign and send transactions, as you will need an account with the native parachain's currency to pay fees when sending transactions.
```

Comment:
```suggestion
    Generated chain TestNet specifications include development accounts "Alice" and "Bob." These accounts are pre-funded with native parachain currency, allowing you to sign and send TestNet transactions. Take a look at the [Polkadot.js Accounts section](https://polkadot.js.org/apps/#/accounts){target=\_blank} to view the development accounts for your chain.
```

Reply:
I think this is where he was going with that? Let me know what you all think. Thanks!